clusters/build-clusters/01_cluster/ci-search/deploy.yaml

```diff
@@ -85,6 +90,9 @@ spec:
         - --bugzilla-url=https://bugzilla.redhat.com/rest
         - --bugzilla-token-file=/etc/bugzilla/api
         - --bugzilla-search=OPEN version:4.14,4.13,4.12,4.11,4.10,4.9,4.8,4.7,4.6,4.5,4.4,4.3,4.2 product="OpenShift Container Platform" AND delta_ts>-2w
+        - --jira-url=https://issues.redhat.com
+        - --jira-token-file=/etc/jira/api
+        - --jira-search="project=OCPBUGS&created>='-14d'&status!='CLOSED'&affectedVersion IN (4.14,4.13,4.12,4.11,4.10,4.9,4.8,4.7,4.6,4.5,4.4,4.3,4.2)"
```

Comment:
Would it be possible to include the TRT project too? We file issues when we're working on a specific test or job problem in our own project.

Reply:
@stbenjam Can that be something we address in a future PR (once we ensure that things are working as expected)?

Reply:
Sure, no problem

Reply:
@stbenjam there's only going to be one jira project for bugs, OCPBUGS.  So TRT would file their issues there.  The bugs will have components so that they can be put onto the appropriate team board views.

cc @thrasher-redhat  to keep me honest.

Reply:
Thats correct.  All OpenShift bugs should be in the `OCPBUGS` project. There might be a few teams who already have bugs in their own projects for legacy reasons, but they aren't currently supported by tooling and won't be supported unless they swap to OCPBUGS.